docker/Dockerfile-jetson

```diff
@@ -28,11 +28,18 @@ ADD https://github.com/ultralytics/assets/releases/download/v8.2.0/yolov8n.pt /u
 # Remove opencv-python from Ultralytics dependencies as it conflicts with opencv-python installed in base image
 RUN grep -v "opencv-python" pyproject.toml > temp.toml && mv temp.toml pyproject.toml
 
+# Download onnxruntime-gpu 1.15 for Jetson devices. Other versions can be seen in https://elinux.org/Jetson_Zoo#ONNX_Runtime
+RUN wget https://nvidia.box.com/shared/static/mvdcltm9ewdy2d5nurkiqorofz1s53ww.whl -O onnxruntime_gpu-1.15.1-cp38-cp38-linux_aarch64.whl
+
 # Install pip packages manually for TensorRT compatibility https://github.com/NVIDIA/TensorRT/issues/2567
 RUN python3 -m pip install --upgrade pip wheel
+RUN pip install onnxruntime_gpu-1.15.1-cp38-cp38-linux_aarch64.whl onnxsim>=0.4.33
 RUN pip install --no-cache tqdm matplotlib pyyaml psutil pandas onnx
 RUN pip install --no-cache -e ".[export]"
 
+# Fix error: `np.bool` was a deprecated alias for the builtin `bool` segmentation error in Tests (onnxruntime-gpu installs it)
+RUN pip install --no-cache numpy==1.23.5
```

Comment:
@hdnh2006 Delete this line because we have a check here:
https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml#L106